packages/flutter_test/lib/flutter_test.dart

```diff
@@ -68,8 +68,10 @@ export 'src/frame_timing_summarizer.dart';
 export 'src/goldens.dart';
 export 'src/image.dart';
 export 'src/matchers.dart';
+export 'src/mock_canvas.dart';
 export 'src/nonconst.dart';
 export 'src/platform.dart';
+export 'src/recording_canvas.dart';
```

Comment:
Is it actually necessary/useful to expose the stuff from recording_canvas as part of the public API for the testing framework? Could that stuff just be a private implementation detail of mock_canvas.dart? Or is it actually used by itself?

Reply:
Looking through some usages, it does actually seem useful. We should enrich its API docs with some sample code to show people how to use it.